sensing/gnss_poser/README.md

```diff
@@ -33,6 +33,7 @@ The `gnss_poser` is a node that subscribes gnss sensing messages and calculates
 | `gnss_frame`         | string | "gnss"           | frame id                                                                                        |
 | `gnss_base_frame`    | string | "gnss_base_link" | frame id                                                                                        |
 | `map_frame`          | string | "map"            | frame id                                                                                        |
+| `coordinate_system`  | int    | "4"              | id number of wanted coordinate system, 4 is a local coordinate system in UTM                    |
```

Comment:
Maybe it is user friendly if the document provides explanation for available coordinate system.

Reply:
I explained all in last commit